src/dsp_tools/utils/connection_live.py

```diff
@@ -336,13 +333,8 @@ def _try_network_action(self, action: Callable[[], Response]) -> Response:
                 time.sleep(2**i)
                 continue
             elif response.status_code != HTTP_OK:
-                raise BaseError(
-                    message="Permanently unable to execute the network action. See logs for more details.",
-                    status_code=response.status_code,
-                    json_content_of_api_response=response.text,
-                    reason_from_api=response.reason,
-                    api_route=response.url,
```

Comment:
The info of these 2 lines has already been logged on line 328 (`self._log_response(response)`)